src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,95 @@
+package calculator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class StringCalculator {
+
+    static int answer = 0;
+
+    public enum Operator {
+        ADD("+"), SUBTRACT("-"), MULTIPLY("*"), DIVIDE("/");
```

Comment:
```suggestion
        ADD("+", (a, b) -> a + b),
        SUBTRACT("-", (a, b) -> a - b),
        MULTIPLY("*", (a, b) -> a * b),
        DIVIDE("/", (a, b) -> a / b);
```
이런식으로 표현해서 관리해주면 어떨까요?
람다식 대신에 Math 관련 메소드를 사용해도 된답니다!